meta/meta-node/src/lib.rs

```diff
@@ -33,11 +33,16 @@ mod ffi {
     }
 
     extern "Rust" {
+        fn connect_block(payload: DmcBlock) -> Result<(), Box<dyn Error>>;
```

Comment:
The signature must be `Result<()>` here because cxx only supports emitting its own result, but we should have the actual error in Rust code.

Reply:
Yup! updated this, still failing.